docs/RunningOnMacOS.md

```diff
@@ -1,6 +1,22 @@
 # Running on MacOS
 
-## With default XCode installation
+## Prerequisites
+
+1. **Node.js**: Ensure you have Node.js 20 or higher installed
+   - Installation options include Homebrew or the official installer from [nodejs.org](https://nodejs.org/)
+
+2. **Xcode Command Line Tools**:
+   - Required for compilers and build tools
+   - Can be installed via `xcode-select --install`
+   - Be sure to accept the Xcode EULA
+
+## Required Configuration Changes
```

Comment:
Small sidenote/Fyi to keep in mind: there's a reason why I called the first section "With default XCode installation" - XCode is not a requirement, people have tried homebrew installed compilers before with CE. I just don't know how that works so did not add "## With homebrew compilers" or something.